control-plane/stor-port/src/types/v0/store/volume.rs

```diff
@@ -630,6 +655,29 @@ impl PublishOperation {
     }
 }
 
+/// SetVolumeProperty Operation parameters.
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+pub struct VolumeProperty {
```

Comment:
I think this can be an enum with the explicit properties we want?

Reply:
Make it enum now.